data/icons/hicolor/32x32/meson.build

```diff
@@ -0,0 +1 @@
+subdir('apps')
```

Comment:
meson.build can be in a directory other than the one were the installed files are -  you can install the 32x32 icon from data/icons/meson.build or even data/meson.build to avoid these extra meson.build that do nothing